content/vault/v1.21.x (rc)/content/docs/commands/secrets/tune.mdx

```diff
@@ -49,6 +52,32 @@ Specify multiple audit non-hmac request keys:
 $ vault secrets tune -audit-non-hmac-request-keys=common_name -audit-non-hmac-request-keys=ttl pki/
 ```
 
+</Tab>
+
+<Tab heading="GUI" group="gui">
+
+> **Note:** Vault 1.21 supports configuration updates (tune) for mounted **keymgmt** secrets engine in the GUI. Support for other engines may be added in future releases.
+
+To tune a keymgmt mount:
```

Comment:
```suggestion
To tune a `keymgmt` mount:
```
For plugin types we typically format it as code unless we write the full name out (e.g., Key management)